### ISSUE TITLE
Accessibility: Missing form label for entry field

### ISSUE THREAD

**Issue by hschnurbusch** [elearning-univie/moodle-mod_wordcloud#9]
Hi

While testing your plugin we found a few issues we wantedt to report. One is reagrding accessibility. If you check the page with the WAVE tool is shows an error for the editor field where users create their entries. The field is missing a form label.

Kind regards
Harald

**Comment by MWRAMC** [comment]
Hi,
thanks for reporting this. We will fix this asap.

Kind regards,
Martin

**Comment by mwreichel** [comment]
Fixed in v.4.0.3.

**Comment by mwreichel** [comment]
Fixed in v.4.0.3.
